packages/nextjs/hooks/scaffold-eth/useAccountBalance.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from "react";
+import { useTargetNetwork } from "./useTargetNetwork";
+import { useQueryClient } from "@tanstack/react-query";
+import { UseBalanceParameters, useBalance, useBlockNumber } from "wagmi";
+
+type UseAccountBalanceParameters = UseBalanceParameters & { watch?: boolean };
+
```

Comment:
Maybe we should add a comment, something like a tiny wrapper around wagmi's `useBalance` hooks which lets you pass watch property or something on that line, similar

Reply:
Good point! [05b67af](https://github.com/scaffold-eth/scaffold-eth-2/pull/829/commits/05b67af858bab47646810cfd95850562268a4dac)